features/pom.xml

```diff
@@ -49,7 +49,9 @@
         - Removed properties that are not used
         - Replaced the value of "geronimo-jms2-spec-version" with "${geronimo-jms_2_spec.version}" to keep it in line with the rest of the project
         - Added netty41 version and removed netty40 version
-        - make camel and artemis qpid-proton-j-bundle-version and qpid-jms-client-version match -->
+        - make camel and artemis qpid-proton-j-bundle-version and qpid-jms-client-version match
+        - Removed CXF specific jackson version in favor of the DDF jackson version
```

Comment:
❗️`cxf.jackson.version` and `jackson2-version` are properties defined in the [CXF](https://github.com/apache/cxf/blob/cxf-3.2.11/parent/pom.xml#L100) and [camel](https://github.com/apache/camel/blob/camel-2.24.2/parent/pom.xml#L334) repos. We try to touch these copied features as little as possible, so I suggest reverting your changes here and only updating the jackson version to match the more-recent ddf jackson version.

```xml
<cxf.jackson.version>${jackson.version}</cxf.jackson.version>
...
<jackson2-version>${cxf.jackson.version}</jackson2-version>
```
and revert the version name changes in the cxf and camel feature files. (spatial-app should use `jackson.version`, s) If this version is 2.9.10 which exactly matches the version in CXF 3.2.11, then you do not leave to leave a comment in the `DDF changes made` list becuase this falls under "Duplicated some properties in the root pom so that they may be used elsewhere in DDF".

Reply:
Done